React/Views/RCTDatePickerManager.m

```diff
@@ -41,5 +41,9 @@ - (UIView *)view
 RCT_EXPORT_VIEW_PROPERTY(onChange, RCTBubblingEventBlock)
 RCT_REMAP_VIEW_PROPERTY(mode, datePickerMode, UIDatePickerMode)
 RCT_REMAP_VIEW_PROPERTY(timeZoneOffsetInMinutes, timeZone, NSTimeZone)
-
+RCT_CUSTOM_VIEW_PROPERTY(textColor, UIColor, RCTDatePicker)
+{
+  UIColor *textColor = json ? [[UIColor alloc] initWithCGColor:[RCTConvert CGColor:json]] : [UIColor blackColor];
+  [view setValue:textColor forKeyPath:@"textColor"];
```

Comment:
This is equivalent to using private APIs, which is reason for rejection in the App Store.